string yaml_path  # filepath of the configuration YAML
---
bool success
string status